### PR TITLE
refactor: switch to remix vite

### PR DESCRIPTION
**Pull request by TrySound**
Ref https://remix.run/docs/en/main/future/vite#setup-vite

## Code Review

- [ ] hi @istarkov, I need you to do
  - conceptual review (architecture, feature-correctness)
  - detailed review (read every line)
  - test it on preview

## Before requesting a review

- [ ] made a self-review
- [ ] added inline comments where things may be not obvious (the "why", not "what")

## Before merging

- [ ] tested locally and on preview environment (preview dev login: 5de6)
- [ ] updated [test cases](https://github.com/webstudio-is/webstudio/blob/main/apps/builder/docs/test-cases.md) document
- [ ] added tests
- [ ] if any new env variables are added, added them to `.env.example` and the `builder/env-check.js` if mandatory

### CODE REVIEW

**Comment by TrySound** [comment]
todo, bump date-ins

File: apps/builder/env.d.ts [hunk 1]
```diff
@@ -0,0 +1,2 @@
+/// <reference types="vite/client" />
```
Comment (istarkov) [inline apps/builder/env.d.ts:1]: why just not add to tsconfig types?

**Review by istarkov — APPROVED** [review]

File: apps/builder/env.d.ts (hunk 1 above)
Comment (TrySound) [reply apps/builder/env.d.ts:1]: I honestly don't know. This is what they recommend to do

**Review by istarkov — APPROVED** [review]

File: apps/builder/env.d.ts (hunk 1 above)
Comment (TrySound) [reply apps/builder/env.d.ts:1]: will fix in another PR

**Review by istarkov — APPROVED** [review]

**Pull request merged**